app/templates/ui/app/detail/detail.module.js

```diff
@@ -3,6 +3,9 @@
 
   angular.module('app.detail', [
     'app.similar',
-    'ui.router'
+    'ui.router',
+    'ml.common', // unit test fails without this
+    'ngToast', // unit test fails without this
+    'ui.bootstrap' // unit test fails without this
```

Comment:
if you change from modal to popover, you probably don't need this, but otherwise:
// Showing a modal for delete confirmation